bundles/org.openhab.binding.mqtt/src/main/java/org/openhab/binding/mqtt/handler/AbstractBrokerHandler.java

```diff
@@ -155,4 +181,51 @@ public void dispose() {
         connectionFuture = new CompletableFuture<>();
         super.dispose();
     }
+
+    public final void registerDiscoveryListener(MQTTTopicDiscoveryParticipant listener, String topic) {
```

Comment:
Could you please add a javadoc to this and the next method? If you are not familiar with the code base it helps a lot for later maintenance.